### PR TITLE
Refactoring/remove react bootstrap

### PR DESCRIPTION
**Pull request by gopienkonikita**
- Removed bootstrap from web: components
- Removed bootstrap from web: common
- Removed bootstrap from web: client
- Removed bootstrap from web: people
- Fixed components styles
- Added base page styles

### CODE REVIEW

File: web/ASC.Web.Components/src/components/modal-dialog/index.js [hunk 1]
```diff
@@ -156,7 +158,11 @@ class ModalDialog extends React.Component {
                 <HeaderText>{headerContent}</HeaderText>
                 {scale ? <CloseButton onClick={onClose}></CloseButton> : ""}
               </Header>
-              <Body>{bodyContent}</Body>
+              <Body>
+                <Text>
```
Comment (AlexeySafronov) [inline web/ASC.Web.Components/src/components/modal-dialog/index.js]: Body supports any elements. You have to revert this changes.

File: web/ASC.Web.Components/src/components/badge/badge.stories.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ storiesOf('Components|Badge', module)
         number={number('number', 10)}
         backgroundColor={color('backgroundColor', '#ED7309')}
         color={color('color', '#FFFFFF')}
-        fontSize={text('fontSize', '11px')}
+        fontSize={number('fontSize', 11)}
```
Comment (andreysavihin) [inline web/ASC.Web.Components/src/components/badge/badge.stories.js]: Use text. Give the ability to specify units of measure.
